rules/power-automate-flows-service-accounts/rule.md

```diff
@@ -15,3 +15,7 @@ Using a service account is a common best practice.
 ::: good
 ![Figure: Good example – see arrows the names are service accounts that will still work after an employee leaves the company](power-platform-services-accounts.png)
 :::
+
+### Do you have issues fixing the lost connections in Power Automate flows?
+
+We have a solution to that problem ✅ Share your flows with SysAdminis, so when a connection string is lost or invalid, SysAdmins will get a notification to fix the problem #MrFixer
```

Comment:
Also explain the pain before:
"When a Flow is only shared with an individual user, if that user is deleted, the flow will be deleted shortly after as well. We don't want deleted flows!"

"Share your flows with SysAdminis" should be "For the above to not happen, share your flow with a group, so it is never deleted when the owner leaves the company or gets deleted". You can use SysAdmins as an example (e.g. Share with the SysAdmin group)

Reply:
Done - Updated the rule 
Thank you